python/src/worker/tasks/cluster_cells.py

```diff
@@ -71,4 +100,9 @@ def compute(self):
         resR = pd.DataFrame(resR)
         resR.set_index("_row", inplace=True)
         resR["cluster"] = pd.Categorical(resR.cluster)
-        return self._format_result(resR)
+
+        # Convert it into a JSON format and patch the API directly
```

Comment:
I assume it's actually an object but in JSON format?

Reply:
we convert python dictionary into json and add it as part of the json patch